.github/workflows/integration_test_ma_finetuner_autotune.yml

```diff
@@ -0,0 +1,33 @@
+name: Integration Test Model Adapter finetuner autotune
```

Comment:
Differences between fintuner & autotune?

Reply:
Before autotune is for hyperparameter tuning, now has been removed.